eng/Generate.ps1

```diff
@@ -185,7 +196,7 @@ if (!($Exclude -contains "TestProjects"))
             continue
         }
         if ($testName.EndsWith("Cadl")) {
-            Add-Cadl $testName $directory
+            Add-CadlRanch-Cadl $testName $directory
```

Comment:
It is not cadl-ranch testServer project, it is the project defined in TestProjects. So it is not right to call Add-CadlRanch-Cadl which will use the cadl file defined in node-modules/@azure-tools/cadl-ranch-specs

Reply:
My plan is to move test\TestProjects\FirstTest-Cadl into test\CadlRanchProjects. What do you think?

Reply:
Yes, I think it is good to separate cadl test projects out, and put them in test\CadlRanchProjects. But still we need to update Add-CadlRanch-Cadl a little bit to refer to right cadl file, for cadl-ranch-testServer-project, it will be node-modules\***, but for other normal cadl ranch project, the cadl file is under the test project directory.